### ISSUE TITLE
After mmgrounding-dino-t model trained my own dataset, the test labels became chaotic.

### ISSUE THREAD

**Issue by h030162** [open-mmlab/mmdetection#12274]
I have defined a great many types of human behaviors, including **sleeping, raising hands, standing, eating, smoking** and so on, more than 20 in all. After I used the mmgrounding-dino-t model to finetune my own dataset, I found that even on the data that had been trained, the inferred labels were in a mess. Moreover, I also found that there were two or more different labels under the same coordinates. What's the cause of this?
The following is a schematic diagram. On the left is the ground truth of the annotation, and on the right is the detection value after the model has been trained.
![aa](https://github.com/user-attachments/assets/4ea7ceba-d891-4438-aaa1-321f065a8dd1)
output is：
{'bbox': [742.95, 368.44, 904.14, 584.8], 'score': 0.55, 'label': 18, 'category': 'raising right'}
{'bbox': [742.95, 368.44, 904.14, 584.8], 'score': 0.48, 'label': 17, 'category': 'raising left'}
However, I only annotated one behavior of sleeping on this picture, and this picture belongs to the training set.
Why does this phenomenon occur?

**Comment by moro0v0** [comment]
I meet problem to train mmgrounding-dino-t in my own dataset,can you show me your config file?

**Comment by moro0v0** [comment]
AND THIS IS MY CONFIG FILE:

_base_ = "./grounding_dino_swin-t_pretrain_obj365.py"
# 我们还需要更改 head 中的 num_classes 以匹配数据集中的类别数
model = dict(
    bbox_head=dict(
        num_classes=4,
    )
)

# 修改数据集相关配置
data_root = '/home/hmc/DUO/'
metainfo = {
    'classes': ('holothurian', 'echinus', 'scallop', 'starfish'),
    # 'palette': [
    #     (220, 20, 60),
    # ]
}
train_dataloader = dict(
    batch_size=8,
    num_workers=8,
    dataset=dict(
        data_root=data_root,
        metainfo=metainfo,
        ann_file='annotations/instances_train.json',
        data_prefix=dict(img='images/train/')))
val_dataloader = dict(
    dataset=dict(
        data_root=data_root,
        metainfo=metainfo,
        ann_file='annotations/instances_test.json',
        data_prefix=dict(img='images/test/')))
test_dataloader = val_dataloader

train_pipeline = [
    dict(type='Resize', scale=(640, 640), keep_ratio=True)
]
test_pipeline = [
    dict(type='Resize', scale=(640, 640), keep_ratio=True)
]
max_epochs = 96
train_cfg = dict(type='EpochBasedTrainLoop', max_epochs=96, val_interval=1)
param_scheduler = [
    dict(
        begin=0,
        by_epoch=True,
        end=96,
        gamma=0.1,
        milestones=[
            11,
        ],
        type='MultiStepLR'),
]
# 修改评价指标相关配置
val_evaluator = dict(ann_file=data_root + 'annotations/instances_test.json')
test_evaluator = val_evaluator

default_hooks = dict(logger=dict(type='LoggerHook', interval=1))

# 使用预训练的 Mask R-CNN 模型权重来做初始化，可以提高模型性能
load_from = "/home/hmc/mmdetection/mmdetection-main/configs/groundingdino_swint_ogc_mmdet-822d7e9d.pth"
